cmd/ingester/app/consumer/rate_limiter.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2018 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package consumer
+
+import (
+	"math"
+	"time"
+)
+
+const cost = 1.0
```

Comment:
What is the purpose of this?

Reply:
Avoiding the use of a magic number throughout. Honestly, I believe 1 is pretty understandable, but we do something very similar in the Go client so I decided to follow that precedent.

Reply:
My bad - I misread